packages/@okta/vuepress-site/docs/guides/terraform-manage-user-access/main/index.md

```diff
@@ -140,7 +140,7 @@ Configure the email authenticator and then create a group, global session policy
 1. Add an `okta_policy_mfa` resource to create an authenticator enrollment policy.
 
    1. Set `status` to `ACTIVE`.
-   1. Set `is_oie` to `true` if you’re using the Okta Identity Engine.
+   1. Set `is_oie` to `true`. Okta Classic supports only setting `okta_password` to `REQUIRED`.
```

Comment:
This is confusing - why are we talking about the `okta_password` setting on the step to set `is_oie`? 

Since this is the first occurrence of a reference to Classic Engine in that doc - it should be Okta Classic Engine. Any subsequent references should just be Classic Engine.

Reply:
Let me know how to proceed... whether it's just the name changes for classic or if one of the modifications above is better.

BTW, I'll be filing a JIRA for updating the sample to be classic friendly in case we get more contract dollars.